features/readme.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Accessing WordPress site
+  As a WordPress developer
+  In order to know this Apache is serving static HTML
+  I'd like to check the WordPress readme.html is visible
+
+  @javascript @insulated
```

Comment:
Yes, though I wouldn't normally include it for this sort of test. `@insulated` will completely reload the browser. It's useful when you don't want tests to bleed into each other because of shared session. That said, it's not necessarily here. I copied and pasted this test from a test project, and I've evidently added it in some point in error. I'll remove it from here.